main.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-restricted-syntax */
+
+localStorage.books = JSON.stringify([]);
+
+function displayBooks() {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.

Reply:
Thanks for the suggestion @rloterh 

I did already used arrow function in my code..
however, i prefer to only use arrow function when am creating `call back` function.
and with normal functions i like to use the `function declaration` way.
that way its more readable.

Thanks again :+1: